contrib/tests/actions/chains/test_quickstart_rules.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 vars:
-        tested_rule: examples.webhook_file
+        tested_rule: examples.examples.webhook_file
```

Comment:
I see.

I'm fine with making this change right now to fix the issue with tests failing. Then after 0.10.0 is out, we can change the name.